features/FxA-45-device-registration-api/README.md

```diff
@@ -0,0 +1,380 @@
+
+  // The public key for encrypting messages to the push endpoint
+  "publicKey": "468601214f60f4828b6cd5d51d9d99d212e7c73657978955f0f5a5b7e2fa1370"
+```
+
+## API
+
+### POST /v1/account/create
+
+#### Requests
+
+##### New device
+
+```
+curl -v \
+-X POST \
+-H "Content-Type: application/json" \
+"https://api-accounts.dev.lcip.org/v1/account/create?keys=true" \
+-d '{
+  "email": "me@example.com",
+  "authPW": "996bc6b1aa63cd69856a2ec81cbf19d5c8a604713362df9ee15c2bf07128efab",
+  "device": {
+    "name": "My Phone",
+    "type": "mobile",
+    "callback": "https://updates.push.services.mozilla.com/update/abcdef01234567890abcdefabcdef01234567890abcdef",
+    "publicKey": "468601214f60f4828b6cd5d51d9d99d212e7c73657978955f0f5a5b7e2fa1370"
+  }
+}'
+```
+
+##### Registered device
+
+```
+curl -v \
+-X POST \
+-H "Content-Type: application/json" \
+"https://api-accounts.dev.lcip.org/v1/account/create?keys=true" \
+-d '{
+  "email": "me@example.com",
+  "authPW": "996bc6b1aa63cd69856a2ec81cbf19d5c8a604713362df9ee15c2bf07128efab",
+  "device": {
+    "id": "0f7aa00356e5416e82b3bef7bc409eef"
+  }
+}'
+```
+
+#### Response
+
+```
+{
+  "uid": "4c352927cd4f4a4aa03d7d1893d950b8",
+  "sessionToken": "27cd4f4a4aa03d7d186a2ec81cbf19d5c8a604713362df9ee15c4f4a4aa03d7d",
+  "keyFetchToken": "7d1893d950b8cd69856a2ec81cbfd7d1893d950b3362df9e56a2ec81cbf19d5c",
+  "authAt": 1392144866,
+  "device": {
+    "id": "0f7aa00356e5416e82b3bef7bc409eef",
+    "name": "My Phone",
+    "type": "mobile",
+    "callback": "https://updates.push.services.mozilla.com/update/abcdef01234567890abcdefabcdef01234567890abcdef",
+    "publicKey": "468601214f60f4828b6cd5d51d9d99d212e7c73657978955f0f5a5b7e2fa1370"
+  }
+}
+```
+
+### POST /v1/account/login
+
+#### Requests
+
+##### New device
+
+```
+curl -v \
+-X POST \
+-H "Host: api-accounts.dev.lcip.org" \
+-H "Content-Type: application/json" \
+-H 'Authorization: Hawk id="d4c5b1e3f5791ef83896c27519979b93a45e6d0da34c7509c5632ac35b28b48d", ts="1373391043", nonce="ohQjqb", hash="vBODPWhDhiRWM4tmI9qp+np+3aoqEFzdGuGk0h7bh9w=", mac="LAnpP3P2PXelC6hUoUaHP72nCqY5Iibaa3eeiGBqIIU="' \
+https://api-accounts.dev.lcip.org/v1/account/device \
+-d '{
+  "name": "My Phone",
+  "type": "mobile",
+  "callback": "https://updates.push.services.mozilla.com/update/abcdef01234567890abcdefabcdef01234567890abcdef",
+  "publicKey": "468601214f60f4828b6cd5d51d9d99d212e7c73657978955f0f5a5b7e2fa1370"
+}'
+```
+
+##### Registered device
+
+```
+curl -v \
+-X POST \
+-H "Host: api-accounts.dev.lcip.org" \
+-H "Content-Type: application/json" \
+-H 'Authorization: Hawk id="d4c5b1e3f5791ef83896c27519979b93a45e6d0da34c7509c5632ac35b28b48d", ts="1373391043", nonce="ohQjqb", hash="vBODPWhDhiRWM4tmI9qp+np+3aoqEFzdGuGk0h7bh9w=", mac="LAnpP3P2PXelC6hUoUaHP72nCqY5Iibaa3eeiGBqIIU="' \
+https://api-accounts.dev.lcip.org/v1/account/device \
+-d '{
+  "id": "0f7aa00356e5416e82b3bef7bc409eef",
+  "name": "My Old Phone"
+}'
+```
+
+#### Response
+
+```
+{
```

Comment:
Is this under the `device` namespace like in [/account/create](https://github.com/mozilla/fxa/pull/69/files#diff-5d7b1f8fd50346a38d4575b2c65aedcaR154)?